docs/source/getting-started/tabular-data/preparing-microdata.ipynb

```diff
@@ -115,6 +115,18 @@
     "`HWUSUAL` column after `.with_columns`."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Select\n",
+    "\n",
+    "[[Polars Documentation](https://docs.pola.rs/user-guide/concepts/expressions-and-contexts/#select)]\n",
+    "\n",
+    "`.select` resolves each passed expression to a column and then returns those columns.\n",
```

Comment:
Now that it's available, should `select` be the default? ie, introduce it first, and use it in all examples (unless `with_columns` is actually needed).

Reply:
`with_columns` does everything `select` does, and more, with no downsides. `select` as a transformation also risks being confused with `select` as a measurement. This is nice to have just because queries are likely to be written with it, including those generated from SQL.